-----------------------------------------------------------------
Revision: a825e87da2ebbdbfb6607c333cbac975d1f8a9d1
Ancestor: e3c8bffcba0e661f54f3be0a71a868b62aeee0fb
Author: nosnilmot@pidgin.im
Date: 2007-06-06T19:28:25
Branch: im.pidgin.pidgin

Modified files:
        pidgin/plugins/gevolution/gevolution.c

ChangeLog: 

don't #include <libebook/e-book-listener.h>, it is apparently an
implementation detail and not part of the libebook API, so we should not
be including it (and it is not installed in recent e-d-s)

============================================================
--- pidgin/plugins/gevolution/gevolution.c	a1a4fffe59cac8aef2723a51c1e663c2e1188731
+++ pidgin/plugins/gevolution/gevolution.c	9c184a8354b352418a5b5c4abee78e5d92a5e1d5
@@ -38,7 +38,6 @@
 
 #include <libedata-book/Evolution-DataServer-Addressbook.h>
 
-#include <libebook/e-book-listener.h>
 #include <libedata-book/e-data-book-factory.h>
 #include <bonobo/bonobo-main.h>
 
